ci/tasks/lint.sh

```
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
+
+cd bosh-cpi-src/src/vsphere_cpi
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)